public/app/plugins/datasource/cloudwatch/syntax.ts

```diff
@@ -0,0 +1,364 @@
+import { Grammar } from 'prismjs';
```

Comment:
So we aren't using the grammar provided by aws at the moment?

Reply:
Not at the moment. I tried integrating it into the work done so far but had some trouble getting it to work